kong/api/routes/upstreams.lua

```diff
@@ -51,11 +50,12 @@ local function post_health(self, db, is_healthy)
   if target then
     local ok, err = db.targets:post_health(upstream, target, is_healthy)
     if not ok then
-      responses.send_HTTP_BAD_REQUEST(err)
+      local body = kong.response.get_default_body(400, err)
```

Comment:
this wasn't converted (a sign that this branch isn't caught by our test suite...)